streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -294,13 +310,15 @@ private void runLoop() {
 
 
         while (stillRunning()) {
+            long now = time.milliseconds();
```

Comment:
Maybe call this `startTime` ("now" is relative -- if it is used later, it is not "now" anymore) ?